Scripts/Notarize.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
```

Comment:
Just a nitpick/FYI: `/bin/sh` is directed to another shell, `/bin/bash` by default (I think, it could also be `/bin/zsh` on a newly installed system, but I don’t know). Moreover, if you are specifically declaring `/bin/sh`, you are constraining yourself to POSIX, which this script is not strictly following (see https://www.shellcheck.net).

Reply:
I removed that.